about/services/education.md

```diff
@@ -0,0 +1,33 @@
+(hub-types:education)=
+# Collaborative learning with data science
+
+:::{admonition} TODO
+This section is incomplete, it needs more information!
+:::
+
+The 2i2c Educational Hubs provide standard data science environments and infrastructure that is meant for teaching data science. They are inspired by 2i2c's experience with the [DataHubs at UC Berkeley](https://docs.datahub.berkeley.edu/en/latest/).
```

Comment:
"....provide standard interactive computing environments and infrastructure designed for teaching data science. These hubs are inspired by 2i2c's experience with the [DataHubs at UC Berkeley](https://docs.datahub.berkeley.edu/en/latest/) and the [Syzygy service](https://syzygy.ca/) for Canada."

Reply:
done 👍